imgui_internal.h

```diff
@@ -1057,7 +1057,7 @@ struct IMGUI_API ImGuiInputTextState
     void        ClearFreeMemory()           { TextW.clear(); TextA.clear(); InitialTextA.clear(); }
     int         GetUndoAvailCount() const   { return Stb.undostate.undo_point; }
     int         GetRedoAvailCount() const   { return STB_TEXTEDIT_UNDOSTATECOUNT - Stb.undostate.redo_point; }
-    void        OnKeyPressed(int key);      // Cannot be inline because we call in code in stb_textedit.h implementation
+    void        OnKeyPressed(ImFont* font, float fontSize, bool useMacOSXBehaviors, int key);      // Cannot be inline because we call in code in stb_textedit.h implementation
 
```

Comment:
Feels like it would be saner to add `ImGuiContext*` as a param here?

Reply:
@ocornut 
Yes, or as you suggested below just having `ImGuiContext*` be an attribute of `ImGuiInputTextState`.